### PR TITLE
Create spot-placement-score.md

### PR DESCRIPTION
**Pull request by avirishuverma**

### CODE REVIEW

File: articles/virtual-machine-scale-sets/spot-placement-score.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+---
+title: Spot Placement Score (Preview)
+description: Learn how to use Azure Spot Placement Score to evaluate deployment success.
+author:
```
Comment (ju-shim) [inline articles/virtual-machine-scale-sets/spot-placement-score.md:4]: Don't forget to add your Microsoft alias to "author" and your GH username to "ms.author".

**Review by ju-shim — APPROVED** [review]
Add yourself as author in the meta tags and you're good to go :)

File: articles/virtual-machine-scale-sets/spot-placement-score.md (hunk 1 above)
Comment (avirishuverma) [reply articles/virtual-machine-scale-sets/spot-placement-score.md]: done, thanks

**Comment by Court72** [comment]
@avirishuverma

Microsoft authors and contributors need to make content contributions through the private repository so the content can be staged and validated by the current validation rules.

After you make your changes in the private repo, review the articles on staging, and fix any validation issues, you can add the sign-off comment to let PR reviewers know that the updates are ready to be merged. 
 
Read about [authors working in the public repo](https://review.docs.microsoft.com/help/contribute/contribute-authors-in-public-repo?branch=main) for more information. 

If you need help getting set up to work in the private repo, see [Get started contributing to the Microsoft Learn platform](https://review.learn.microsoft.com/help/get-started/?branch=main&tabs=learn).

We'll close this PR. Thank you.

**Pull request closed without merging**